src/elements/Tabs/TabsContainer.tsx

```diff
@@ -19,21 +23,77 @@ interface Indicator {
   Component: React.FC<TabItemProps<string>>
 }
 
+interface PillTabItemProps extends TabItemProps<string> {
+  onTabPress?: (tabName: string) => void
+  focusedTab: { value: string }
+}
+
+const PillTabItem: React.FC<PillTabItemProps> = (props) => {
```

Comment:
very minor: Most of the time we destructure the props, which I think makes the code easier to read 🤷

```suggestion
const PillTabItem: React.FC<PillTabItemProps> = ({ focusedTab, .... } ) => {
```

Reply:
this was actually intentional because we have `focuedTab` in state.
<img width="605" alt="Screenshot 2025-04-28 at 17 38 26" src="https://github.com/user-attachments/assets/01ce03cf-ea81-4b2b-b754-727792732834" />

I will use `focuedTabProp` instead